src/kibana/plugins/vis_types/controls/point_series_options.html

```diff
@@ -0,0 +1,42 @@
+<div>
+  <div class="vis-option-item" ng-show="vis.hasSchemaAgg('segment', 'date_histogram')">
+    <label>
+      <input type="checkbox" ng-model="vis.params.addTimeMarker" ng-checked="vis.params.addTimeMarker">
+      Current time marker
+    </label>
+  </div>
+  <div class="vis-option-item">
+    <label>
+      <input type="checkbox" ng-model="vis.params.setYExtents">
+      Set Y-Axis Extents
+    </label>
+    <div ng-if="vis.params.setYExtents">
+      <label>
+        y-max
+        <input name="yMax"
+               class="form-control"
+               type="number"
+               step="0.1"
+               min="{{vis.params.yAxis.min}}"
+               ng-model="vis.params.yAxis.max"
+               ng-required="vis.params.setYExtents">
+      </label>
+      <label>
+        y-min
+        <input name="yMin"
+               class="form-control"
+               type="number"
+               step="0.1"
+               max="{{vis.params.yAxis.max}}"
```

Comment:
Its important we keep validation consistent. Its a known issue in Angular that `min/max="{{x.y.z}}"` does not work. It might be fixed in Angular 1.3, but for now we need to do form validation consistently, which means directive based validators right now.